include/bbp/sonata/report_reader.h

```diff
@@ -27,6 +27,7 @@ struct SONATA_API DataFrame {
 
 using Spike = std::pair<NodeID, double>;
 using Spikes = std::vector<Spike>;
+using SpikesArrays = std::pair<std::vector<NodeID>, std::vector<double>>;
```

Comment:
I'd use a `struct` here to make it more future proof; if we decide to add additional info, the calling code can be recompiled without changes.

perhaps:
```
struct SpikeTimes{
  std::vector<NodeID> nodeIds;
  std::vector<double> timestamps;
};
```